Sources/MapboxCoreNavigation/NavSDK.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import MapboxDirections
+
+public struct NavSDK {
```

Comment:
Consider combining this struct with NavigationSettings.

Reply:
done, please review the latest revision.